### PR TITLE
update cget

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'PythonBundle'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb:7]:
```suggestion
easyblock = 'PythonPackage'
```

File: easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'cget'
+version = '0.2.0'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb:15]: I don't think this really needs to be at `foss` level, we can probably drop this to `GCCcore`
```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

File: easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb:16]:
```suggestion

sources = [SOURCE_TAR_GZ]

```

File: easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('cget', version, {
+        'checksums': ['835009ba6d623a36eee8056975d7cdbeebb0e0091a058b572ed433fb12ae18e8'],
+    }),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cget/cget-0.2.0-foss-2020b.eb:30]:
```suggestion
download_dep_fail = True
]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]
Since `exts_list` now consists of only the `cget` package, I've made some suggestions that switch to `PythonPackage` instead of `PythonBundle`.
I would also drop the toolchain to `GCCcore` (the filename should be updated to match)

**Comment by fizwit** [comment]
PR 13277 is open for cget-0.1.9 for GCCcore 10.2.0 . Changed my PR to GCCcore 11.2 and version 0.2.0.  Looks like the tests were failing due to no santiy_check_paths.

File: easybuild/easyconfigs/c/cget/cget-0.2.0-GCCcore-11.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+version = '0.2.0'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['835009ba6d623a36eee8056975d7cdbeebb0e0091a058b572ed433fb12ae18e8']
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/cget/cget-0.2.0-GCCcore-11.2.0.eb:19]: this should make the test suite happier
```suggestion

builddependencies = [('binutils', '2.35')]

```

**Pull request closed without merging**